dictionary/index.js

```
@@ -0,0 +1,39 @@
+
+const { Validation, logger } = require('../src/utils');
+const { ErrorWithMessage } = require('../src/utils/ErrorWithMessage');
+
+const locales = {
+    wallet: require('./wallet.local'),
+    common: require('./common.local'),
+}
+
+const setLocale = (provider) => {
+    if (Validation.isServiceProvider(provider)) {
+        const filename = provider + '.local.js';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)